sdk/src/androidTest/java/com/bugsnag/android/UserTest.java

```diff
@@ -0,0 +1,81 @@
+package com.bugsnag.android;
+
+import android.content.SharedPreferences;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.filters.SmallTest;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.UUID;
+
+import static com.bugsnag.android.BugsnagTestUtils.getSharedPrefs;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.nullValue;
+import static org.hamcrest.core.Is.is;
+
+@RunWith(AndroidJUnit4.class)
+@SmallTest
+public class UserTest {
+    @Test
+    public void testBuilder_defaults() {
+        User user = User.builder().build();
+
+        assertThat(user.getId(), is(nullValue()));
+        assertThat(user.getEmail(), is(nullValue()));
+        assertThat(user.getName(), is(nullValue()));
+    }
+
+    @Test
+    public void testBuilder_values() {
```

Comment:
Our codestyle uses camelcase, looks like I've got a bit of work to do configuring our checkstyle as it should have caught this D=

Reply:
Have you considered changing it? I think
`testMethodA_defaults` and `testMethodA_nullArgument`
is more readable than
`testMethodADefaults` and `testMethodANullArguments`
because it's immediately clear that it's two tests for the same method, just differents aspects.

I'll change it though, I don't have strong feelings about this.